advocacy_docs/edb-postgres-ai/databases/postgres_distributions/advanced_security.mdx

```diff
@@ -0,0 +1,26 @@
+| Data redaction                             | No                                 | Yes                                | Yes                                                                                                                                             | Yes                                                                                                                                                          |
+| Profile and password management            | No                                 | Yes                                | Yes                                                                                                                                             | Yes                                                                                                                                                          |
+| SQL injection protection                   | No                                 | Yes                                | Yes                                                                                                                                             | Yes                                                                                                                                                          |
+| Stored procedure source code obfuscation   | No                                 | Yes                                | Yes                                                                                                                                             | Yes                                                                                                                                                          |
+| Virtual private databases                  | No                                 | Yes                                | Yes                                                                                                                                             | Yes                                                                                                                                                          |
+| Statement audit logging                    | [pgAudit](/pg_extensions/pgaudit/) | [pgAudit](/pg_extensions/pgaudit/) | [EDB Audit](/epas/latest/database_administration/01_configuration_parameters/03_configuration_parameters_by_functionality/07_auditing_settings/) | [EDB Audit](/epas/latest/database_administration/01_configuration_parameters/03_configuration_parameters_by_functionality/07_auditing_settings/01_edb_audit) |
+| Role and priviledge usage analysis         | No                                 | Yes                                | Yes                                                                                                                                             | Yes                                                                                                                                                          |
```

Comment:
For EPAS we can link to DBMS package, for PGE we can link to upstream Postgresql. 
https://www.enterprisedb.com/docs/epas/latest/epas_security_guide/03_virtual_private_database/